src/lib/env-vars.ts

```diff
@@ -194,30 +192,3 @@ export const checkEnvVarValueValidity = (value: string) => {
 		);
 	}
 };
-
-export interface EnvVar {
-	name: string;
-	value: string;
-}
-
-export const mergeEnvVars = (
```

Comment:
This is used by the /environment endpoint in cloud API. Are you moving it out of here or completely removing it?

Reply:
Removing it completely from open source because it's unused here and is purely used for very very old devices